DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -48,58 +48,190 @@
 		</orientationSpecializedModes606>
 	</Enums>
 	<ActionIDs>
-		<ActionID ID="75" name="PlaySoundByID()">
+		<ActionID ID="34" name="Position and Rotation (Ignore the Pummy Point)">
+			<Properties1>
+				<field type="[S]" arg="" name="X Rotation Speed"/>
+				<field type="[S]" arg="" name="X Rotation Speed Multiplier"/>
+				<field type="[S]" arg="" name="Y Rotation Speed"/>
+				<field type="[S]" arg="" name="Y Rotation Speed Multiplier"/>
+				<field type="[S]" arg="" name="Z Rotation Speed"/>
+				<field type="[S]" arg="" name="Z Rotation Speed Multiplier"/>
+=			</Properties1>
+			<Fields1>
+				<Field name="" dataType="s32" isBool=""/>
+			</Fields1>
+		</ActionID>
+		<ActionID ID="35" name="Position and Rotation (Ignore the Pummy Point)">
 			<Fields1>
-				<Field name="Sound ID" dataType="s32" />
-				<Field name="Volume" dataType="f32" />
+				<Field name="X Position" dataType="f32"/>
+				<Field name="Y Position" dataType="f32"/>
+				<Field name="Z Position" dataType="f32"/>
+				<Field name="X Rotation" dataType="f32"/>
+				<Field name="Y Rotation" dataType="f32"/>
+				<Field name="Z rotation" dataType="f32"/>
+			</Fields1>
+		</ActionID>
+		<ActionID ID="36" name="Random Position and Rotation?">
+			<Fields1>
+				<Field name="X Position Max?" dataType="f32"/>
+				<Field name="Y Position Max?" dataType="f32"/>
+				<Field name="Z Position Max?" dataType="f32"/>
+				<Field name="X Rotation Max?" dataType="f32"/>
+				<Field name="Y Rotation Max?" dataType="f32"/>
+				<Field name="Z rotation Max?" dataType="f32"/>
+				<Field name="X Position Min?" dataType="f32"/>
+				<Field name="Y Position Min?" dataType="f32"/>
+				<Field name="Z Position Min?" dataType="f32"/>
+				<Field name="X Rotation Min?" dataType="f32"/>
+				<Field name="Y Rotation Min?" dataType="f32"/>
+				<Field name="Z rotation Min?" dataType="f32"/>
+			</Fields1>
+		</ActionID>
+		<ActionID ID="75" name="Play Sound by ID">
+			<Fields1>
+				<Field name="Sound ID" dataType="s32"/>
+				<Field name="Volume" dataType="f32"/>
 				<Field name="Repeat" dataType="s32" isBool=""/>
 			</Fields1>
 		</ActionID>
-		<ActionID ID="131" name="EffectModifier()">
+		<ActionID ID="105" name="Brownian Particle Motion?">
```

Comment:
No clue what brownian particle motion is, but it seems unclear. There's probably a more straight forward way to call it